packages/trader/src/AppV2/Components/AccumulatorStats/accumulator-stats-description.tsx

```diff
@@ -1,22 +1,14 @@
 import React from 'react';
 import { ActionSheet, Text } from '@deriv-com/quill-ui';
 import { Localize } from '@deriv/translations';
-import { DESCRIPTION_VIDEO_ID, UNIFIED_MODE_VIDEO_ID } from 'Modules/Trading/Helpers/video-config';
+import { UNIFIED_MODE_VIDEO_ID } from 'Modules/Trading/Helpers/video-config';
+import StreamIframe from '../OnboardingGuide/StreamIframe';
```

Comment:
should StreamIframe be outside OnboardingGuide

Reply:
Agree, will remove